Classes/GraphQL/Resolver/Type/MutationResolver.php

```diff
@@ -435,6 +445,86 @@ public function uploadFiles($_, array $variables): array
         return $results;
     }
 
+    /**
+     * Replaces an asset and its usages
+     *
+     * @param $_
+     * @param array $variables
+     * @return array<array>
+     */
+    public function replaceAsset($_, array $variables, AssetSourceContext $assetSourceContext): ?array
```

Comment:
This mainly reuses the old `Neos\Media\Browser\Controller\updateAssetResourceAction`